### PR TITLE
8.1 updates

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

File: hardware/tftp/Android.mk [hunk 1]
```diff
@@ -5,25 +5,25 @@ $(shell mkdir -p $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR)/rfs/msm/mpss/readonly/
...
-$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -s /firmware rfs/msm/slpi/readonly/firmware && popd > /dev/null)
-$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -s /odm/firmware rfs/msm/slpi/readonly/vendor/firmware && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /data/tombstones/lpass rfs/msm/slpi/ramdumps && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /persist/rfs/msm/slpi rfs/msm/slpi/readwrite && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /persist/rfs/shared rfs/msm/slpi/shared && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /persist/hlos_rfs/shared rfs/msm/slpi/hlos && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /firmware rfs/msm/slpi/readonly/firmware && popd > /dev/null)
+$(shell pushd $(PRODUCT_OUT)/$(TARGET_COPY_OUT_VENDOR) > /dev/null && ln -sf /odm/firmware rfs/msm/slpi/readonly/vendor/firmware && popd > /dev/null)
```
Comment (jamuir) [inline hardware/tftp/Android.mk]: we really need a macro for sym-link creation :-)

Comment (alviteri) [reply hardware/tftp/Android.mk]: ad every dirty build (not clean one) there is a fastidious warning about it ....

**Comment by alviteri** [comment]
this can be merged ... @jerpelea I doubt google will update sdm845 HAL for oreo ...

**Comment by jerpelea** [comment]
please rebase

**Pull request closed without merging**